exporter.go

```diff
@@ -25,7 +25,7 @@ type WmiCollector struct {
 }
 
 const (
-	defaultCollectors = "cpu,cs,logical_disk,net,os,system"
+	defaultCollectors = "cpu,cs,logical_disk,net,os,service,system"
```

Comment:
@carlpett Do you feel this should be enabled by default? If so, patch looks good to me. If not, please change this :)